### PR TITLE
Switch from webpack to vite

### PR DESCRIPTION
**Pull request by bmatthieu3**
This branch is an attempt to switch from webpack to vite.
After merging this branch there will be:
* Aladin lite will be contained into a single file of ~800kB gzipped
* There is a ES module type one, allowing you to use aladin lite with imports notations. It may be easier to deploy a npm package and use it like so:
```js
<div id="aladin-lite-div" style="width:400px;height:400px;"></div>
<script type="module">
import A from 'aladin-lite';
let aladin;
A.init.then(() => {
    aladin = A.aladin('#aladin-lite-div', {survey: "P/DSS2/color", fov:60});
});
</script>
```
And there is a UMD module type version of aladin lite, for using aladin lite in a more standard/traditionnal way
```js
<div id="aladin-lite-div" style="width:400px;height:400px;"></div>
<!-- URL to define -->
<script type="text/javascript" src="https://aladin.cds.unistra.fr/AladinLite/api/v3/latest/aladin.umd.cjs" charset="utf-8"></script>
<script type="text/javascript">
let aladin;
A.init.then(() => {
    aladin = A.aladin('#aladin-lite-div', {survey: "P/DSS2/color", fov:60});
});
</script>
```

### CODE REVIEW

**Comment by bmatthieu3** [comment]
I refer to the issues: #77 , #64. I encourage people interested to have a look, I am very new to this and surely not conscient of all the vite optimizations and npm package tricks.

**Comment by diego-ge** [comment]
This looks awesome, I was comparing with the changes I did to compile using Vite and we have pretty much the same.
Also I have tested your compiled dist directory in a new raw project to check if it will complain if its installed via npm and it worked like a charm: :clap:

**Comment by simontorres** [comment]
This is amazing! Thank you very much for working on this. This will benefit a lot of users.

**Comment by cquiroz** [comment]
Thanks a lot @bmatthieu3 this is absolutely great

File: package.json [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
+  "main": "./dist/aladin.js",
   "files": [
-    "dist/*"
+    "dist/aladin.js",
+    "dist/aladin.umd.cjs",
+    "dist/index.html"
   ],
+  "exports": {
+    ".": {
+      "import": "./dist/aladin.js"
```
Comment (diego-ge) [inline package.json:18]: @bmatthieu3 you could add a second option to use umd file by default as mentioned in [conditional exports](https://nodejs.org/api/packages.html#conditional-exports).
`"require": "./dist/aladin.umd.cjs"`

Comment (bmatthieu3) [reply package.json:18]: done, do not hesitate if there are other things

**Pull request merged**